corehq/apps/userreports/filters/__init__.py

```diff
@@ -30,7 +32,7 @@ def __init__(self, filters):
         assert len(self.filters) > 0
 
     def __call__(self, item, context=None):
-        return all(filter(item, context) for filter in self.filters)
+        return all(list(filter(item, context)) for filter in self.filters)
```

Comment:
This doesn't look right. `filter` is a local here, not the global/builtin `filter`. Does it return a list?